hw04-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,18 @@
+package homework;
+
+
+import java.util.*;
+
+public class CustomerReverseOrder {
+    //todo: 2. надо реализовать методы этого класса
+    //надо подобрать подходящую структуру данных, тогда решение будет в "две строчки"
+
+    Deque<Customer> _queue = new LinkedList<Customer>();
```

Comment:
Поля всегда приватные:
```suggestion
    private Deque<Customer> _queue = new LinkedList<Customer>();
```